apps/web/styles/globals.css

```diff
@@ -2,7 +2,37 @@ body {
   background-color: black;
 }
 
-input[type=number]::-webkit-inner-spin-button, 
+input[type=number]::-webkit-inner-spin-button,
 input[type=number]::-webkit-outer-spin-button {
   opacity: 0;
 }
+
+.teamsImage {
+  width: 400px;
+  height: 300px;
+}
+
+@media (max-width: 600px) {
+  .teamsImage {
+    width: 250px;
+    height: 200px;
+  }
+}
+
+::-webkit-scrollbar {
+  width: 6px;
+  height: 6px;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #2836484D;
+  border-radius: 3px;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: #28364880;
+}
+
+::-webkit-scrollbar-corner {
+  background: transparent;
+}
```

Comment:
Here the color will be static across dark/light theme. It would be better to move this part inside the mui (data table) style overwrites as we have the global scrollbar also there. And the only thing that we might need to overwrite in the data table is the width of the scroller as the other properties are inherited from the global style.

Reply:
we should also include this one since the corners looked like a white square without it
```
::-webkit-scrollbar-corner {
  background: transparent;
}
```